docs/notebooks/kinetic_scheme_visualization.ipynb

```
@@ -0,0 +1,684 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from glotaran.testing.simulated_data.sequential_spectral_decay import SCHEME as SCHEME_SEQ\n",
+    "from glotaran.testing.simulated_data.parallel_spectral_decay import SCHEME as SCHEME_PAR\n",
+    "from pyglotaran_extras import show_kinetic_scheme, show_dataset_kinetic_scheme, KineticSchemeConfig\n",
```

Ruff (0.15.0)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)